src/Serilog/Configuration/LoggerDestructuringConfiguration.cs

```diff
@@ -147,7 +149,23 @@ public LoggerConfiguration ByTransforming<TValue>(Func<TValue, object> transform
         public LoggerConfiguration ToMaximumDepth(int maximumDestructuringDepth)
         {
             if (maximumDestructuringDepth < 0) throw new ArgumentOutOfRangeException(nameof(maximumDestructuringDepth));
-            _setMaximumDepth(maximumDestructuringDepth);
+            _setMaximumDepth(maximumDestructuringDepth, false);
+            return _loggerConfiguration;
+        }
+
+        /// <summary>
+        /// When destructuring objects, depth will be limited to 5 property traversals deep to
```

Comment:
The default depth is actually 10. See updated summary for the [`ToMaximumDepth(int maximumDestructuringDepth)`](https://github.com/serilog/serilog/blob/dev/src/Serilog/Configuration/LoggerDestructuringConfiguration.cs#L139) overload on dev.

Reply:
I've updated the summary to match the latest version